src/modules/storagefacility/TableComponent.js

```diff
@@ -625,4 +533,57 @@ const props = {
   emitSuccess
 };
 
-export default inject(data, commands, props)(StorageUnitsContainer);
+const processBarcode = (barCode, props) => {
```

Comment:
prcoessBarcode can be out in Utils from here and picklist. Also we can reflector it into reusable methods.